### PR TITLE
fix: Add displayMinutes option to time component

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
fixes: https://github.com/SAP/fundamental-ngx/issues/1084
#### Please provide a brief summary of this pull request.
I added `displayMinutes` option to time, timePicker, datetimePicker component, which controls the display of minuts input.
#### If this is a new feature, have you updated the documentation?
The documentation is updated by 2 examples on timePicker and time components.

### CODE REVIEW

**Comment by MattL75** [comment]
Might as well add it for hour while you're at it. I'm not sure I like this input-based solution but since it was there to begin with, I guess we can keep going forward with it.

**Comment by JKMarkowski** [comment]
Hours disabling added.

File: docs/app/documentation/component-docs/time-picker/time-picker-docs.component.html [hunk 1]
```diff
@@ -1,63 +1,74 @@
...
 <component-example [name]="'ex3'">
-  <fd-time-picker-no-seconds-example></fd-time-picker-no-seconds-example>
+    <fd-time-picker-no-seconds-example></fd-time-picker-no-seconds-example>
 </component-example>
 <code-example [exampleFiles]="timePickerNoSeconds"></code-example>
 
+<separator></separator>
+<h2>Time Picker Without Minutes and Seconds</h2>
+<description>Use <code>[displaySeconds]="false"</code>, <code>[displayMinutes]="false"</code> or <code>[displayHours]="false"</code>
+    to turn off seconds, minutes or hours for the time picker. At least one of them should be enabled.
```
Comment (MattL75) [inline docs/app/documentation/component-docs/time-picker/time-picker-docs.component.html]: capital t

File: docs/app/documentation/component-docs/time/time-docs.component.html [hunk 2]
```diff
@@ -37,11 +37,22 @@ <h2>Clock Without Seconds</h2>
 
 <separator></separator>
 
+<h2>Clock Without Minutes and Seconds</h2>
+<description>Use <code>[displaySeconds]="false"</code>, <code>[displayMinutes]="false"</code> or <code>[displayHours]="false"</code>
+    to turn off seconds, minutes or hours for the time picker. At least one of them should be enabled.
```
Comment (MattL75) [inline docs/app/documentation/component-docs/time/time-docs.component.html]: capital t

File: library/src/lib/time/time.component.ts [hunk 3]
```diff
@@ -38,6 +38,16 @@ export class TimeComponent implements OnChanges, ControlValueAccessor {
     /** @Input When set to false, hides the input for seconds. */
     @Input() displaySeconds: boolean = true;
 
+    /** @Input When set to false, hides the input for minutes. */
```
Comment (MattL75) [inline library/src/lib/time/time.component.ts]: can you remove all these `@Input`? It's leftover from the previous way docs were built

**Comment by droshev** [comment]
@JKMarkowski can you resolve the conflicts.

**Pull request merged**